packages/next/shared/lib/router/router.ts

```diff
@@ -1137,6 +1139,13 @@ export default class Router implements BaseRouter {
     window.history.back()
   }
 
+  /**
+   * Go forward in history
+   */
+  forward() {
+    window.history.forward()
```

Comment:
If we add this, we might want to document in https://nextjs.org/docs/api-reference/next/router#router-object

Reply:
This was technically not exposed on the public interface made over here:

https://github.com/vercel/next.js/blob/dc3cd950174ff22ebb6d9d62a0cfad6f96b61cae/packages/next/client/router.ts#L160

So it won't be available via the `useRouter` hook or `withRouter` hoc from `next/router`.